packages/cli/package.json

```diff
@@ -30,7 +30,6 @@
     "scripts/preinstall.js"
   ],
   "ava": {
-    "compileEnhancements": false,
```

Comment:
Looks like it was supposed to be nested under `babel` https://github.com/avajs/babel#disable-enhancements